osu.Game/Screens/Play/BeatmapMetadataDisplay.cs

```diff
@@ -66,11 +72,11 @@ private void load()
                     Direction = FillDirection.Vertical,
                     Children = new[]
                     {
-                        facade.With(d =>
+                        logoFacade?.With(d =>
                         {
                             d.Anchor = Anchor.TopCentre;
                             d.Origin = Anchor.TopCentre;
-                        }),
+                        }) ?? Drawable.Empty(),
```

Comment:
class name qualifier is redundant (but i'd like this to be reverted anyhow, see previous comment)

```suggestion
                        }) ?? Empty(),
```